### PR TITLE
docs: inspector component details for configuration feature

### PR DESCRIPTION
**Pull request by rossb220**
Just adding a comment regarding the `Inspector` component. If this doesn't exist on a custom layout the configuration feature does not seem to work i.e the popup fails to appear

### CODE REVIEW

**Comment by fzaninotto** [comment]
Thanks! Could you please add an example of a custom layout with embedded Inspector in this new section?

File: docs/Configurable.md [hunk 1]
```diff
@@ -214,3 +214,53 @@ const MyAppBar = () => (
     </AppBar>
 );
 ```
+
+## `<Inspector>`
+
+The `<Inspector>` component exists within the layouts provided by react-admin. If you are using a custom layout then you should add the `<Inspector>` component within your layout
```
Comment (slax57) [inline docs/Configurable.md:220]:
```suggestion
The `<Inspector>` is already included in the layouts provided by react-admin. If you are using a custom layout, you need to add the `<Inspector>` component to your layout.
```

File: docs/Configurable.md [hunk 2]
```diff
@@ -214,3 +214,53 @@ const MyAppBar = () => (
...
+## `<Inspector>`
+
+The `<Inspector>` component exists within the layouts provided by react-admin. If you are using a custom layout then you should add the `<Inspector>` component within your layout 
+
+{% raw %}
+```jsx
+// in src/MyLayout.js
+import * as React from 'react';
+import { Box } from '@mui/material';
+import { AppBar, Menu, Sidebar, InspectorButton } from 'react-admin';
```
Comment (slax57) [inline docs/Configurable.md:227]:
```suggestion
import { AppBar, Menu, Sidebar, Inspector } from 'react-admin';
```

File: docs/Configurable.md [hunk 3]
```diff
@@ -214,3 +214,53 @@ const MyAppBar = () => (
...
+                    flexGrow={2}
+                    p={3}
+                    marginTop="4em"
+                    paddingLeft={5}
+                >
+                    {children}
+                </Box>
+            </Box>
+        </Box>
+        <InspectorButton />
```
Comment (slax57) [inline docs/Configurable.md:260]:
```suggestion
        <Inspector />
```

File: docs/WrapperField.md [hunk 4]
```diff
@@ -5,7 +5,8 @@ title: "The WrapperField Component"
 
 # `<WrapperField>`
 
-This component simply renders its children. Why would you want to use such a dumb component? To combine several fields in a single cell (in a `<Datagrid>`) or in a single row (in a `<SimpleShowLayout>`). 
+This component simply renders its children. Why would you want to use such a dumb component? To combine several fields in a single cell (in a `<Datagrid>`) or in a single row (in a `<SimpleShowLayout>`) or 
+a group of form fields (in a `<SimpleFormConfigurable>`)
```
Comment (slax57) [inline docs/WrapperField.md:9]:
```suggestion
This component simply renders its children. Why would you want to use such a dumb component? To combine several fields in a single cell (in a `<Datagrid>`), in a single row (in a `<SimpleShowLayout>`) or in a group of inputs (in a `<SimpleFormConfigurable>`) .
```

File: docs/WrapperField.md [hunk 5]
```diff
@@ -25,4 +26,25 @@ const BookList = () => (
...
+            <TextInput source="title"/>
+            <WrapperField source="author">
+                <Stack>
+                    <TextInput source="author_first_name"/>
+                    <TextInput source="author_last_name"/>
+                </Stack>
+            </WrapperField>
+        </SimpleFormConfigurable>
+    </Edit>
+);
```
Comment (slax57) [inline docs/WrapperField.md:46]:
```suggestion
import { Edit, WrapperField, TextInput, SimpleFormConfigurable } from 'react-admin';
import { Stack } from '@mui/material';

const PostEdit = () => (
    <Edit>
        <SimpleFormConfigurable>
            <TextInput source="title"/>
            <WrapperField source="author">
                <Stack>
                    <TextInput source="author_first_name"/>
                    <TextInput source="author_last_name"/>
                </Stack>
            </WrapperField>
        </SimpleFormConfigurable>
    </Edit>
);
```

**Review by slax57 — CHANGES_REQUESTED** [review]

**Comment by slax57** [comment]
Thanks for the changes.

Unfortunately it seems I do not have the permissions to commit on your branch, so unfortunately I can't commit the suggested changes myself.
Can you please accept them?

Thanks.

**Comment by rossb220** [comment]
@slax57 That's been done but would you like me to squash the commits?

**Comment by slax57** [comment]
No need I'll do it when merging. Thanks a lot! :pray:

**Review by slax57 — APPROVED** [review]

**Pull request merged**